packages/strategies-keep3r/contracts.json

```diff
@@ -75,6 +76,12 @@
       "keep3r": {
         "address": "0x1cEB5cB57C4D4E2b2433641b95Dd330A33185A44"
       },
+      "keep3rV2": {
+        "address": "0x4a6cff9e1456eaa3b6f37572395c6fa0c959edab"
```

Comment:
@TheAustrian1998 watch out current keep3r address is `0xeb02addCfD8B773A5FFA6B9d1FE99c566f8c44CC`

Reply:
thanks both!